applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -135,6 +135,7 @@ The ``<shutdown_mode>`` parameter accepts only the following integer values:
 * ``0`` - Enter Idle.
   You can also use the syntax ``AT#XSLEEP``.
 * ``1`` - Enter Sleep.
+* ``2`` - Power off UART.
 
 The default value is 0.
```

Comment:
Need to add the behavior for power off to the list in line 146/147,

Reply:
Updated